git/git.go

```diff
@@ -80,6 +96,19 @@ func ResolveRef(ref string) (*Ref, error) {
 	return fullref, nil
 }
 
+func ResolveRefs(refnames []string) ([]*Ref, error) {
+	refs := make([]*Ref, len(refnames))
+	for i, name := range refnames {
+		ref, err := ResolveRef(name)
+		if err != nil {
+			return refs, err
+		}
+
+		refs[i] = ref
+	}
+	return refs, nil
+}
```

Comment:
It already was a loop: https://github.com/github/git-lfs/blob/2cf082fa14a738f73db6d172eba800ca118e6d97/commands/command_fetch.go#L46-L51

I extracted this function, and left this note to work on after the weekend. Guess I could've said as much on my msg from Friday :)